Parsers/ASimAuditEvent/Parsers/ASimAuditEventGithub.yml

```diff
@@ -0,0 +1,72 @@
+Parser:
+  Title: ASIM Audit Event parser for <product name>
+  Version: '0.1'
+  LastUpdated: <Mar 27, 2024>
+Product:
+  Name: Github
+Normalization:
+  Schema: AuditEvent
+  Version: '<current schema version>'
+References:
+- Title: ASIM Audit Event Schema
+  Link: https://aka.ms/ASimAuditEventDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing the <product name> logs to the ASIM Audit Event normalized schema.
+ParserName: _ASim_AuditEvent_Github
+EquivalentBuiltInParser: _ASim_AuditEvent_Github
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+     let parser = (
+     disabled:bool = false
+     )
+     {
+     let EventResultDetailslookup=datatable(EventOriginalResultDetails:string,EventResult:string, EventType:string )
```

Comment:
could there be additional 'action' apart from what we have in this table? did you check product documentation?